perllib/FixMyStreet/App/Controller/Waste/Bulky.pm

```diff
@@ -59,6 +59,21 @@ sub item_list : Private {
 
     my $max_items = $c->cobrand->bulky_items_maximum;
     my $field_list = [];
+
+    my $notes_field = {
+        type => 'TextArea',
+        label => 'Item note (optional)',
+        maxlength => 100,
+        tags => { hint => 'Describe the item to help our crew pick up the right thing' },
+    };
+    if ($c->cobrand->moniker eq 'brent') {
+        $notes_field = {
+            type => 'Text',
+            label => 'If other small electrical, please specify',
```

Comment:
I think this only happens on 'other' so not a big deal but could say please give a description of the 'other' item

Reply:
I worded it like that in case the JS doesn't load/work, and so it'd be displayed all the time, and couldn't face having the text update :)